xarray/tests/test_backends.py

```diff
@@ -5933,23 +5879,23 @@ def test_encode_zarr_attr_value() -> None:
 @requires_zarr
 def test_extract_zarr_variable_encoding() -> None:
     var = xr.Variable("x", [1, 2])
-    actual = backends.zarr.extract_zarr_variable_encoding(var, zarr_format=3)
+    actual = backends.zarr.extract_zarr_variable_encoding(var)
     assert "chunks" in actual
     assert actual["chunks"] == ("auto" if has_zarr_v3 else None)
 
     var = xr.Variable("x", [1, 2], encoding={"chunks": (1,)})
-    actual = backends.zarr.extract_zarr_variable_encoding(var, zarr_format=3)
+    actual = backends.zarr.extract_zarr_variable_encoding(var)
```

Comment:
this is needed, let's undo this change

Reply:
I don't really know where this is from... I merged main branch into this PR and did a `git pull --rebase origin pydap4` . Totally happy to change that.

Reply:
feel free to just merge, no rebase is necessary if that makes it easier

Reply:
fixed